### PR TITLE
Exocomp Factory

### PR DESCRIPTION
**Pull request by WebSpider**

### CODE REVIEW

File: buildings/exocomp-factory.json [hunk 1]
```diff
@@ -0,0 +1,1824 @@
...
+          "name": "Operations",
+          "level": 16
+        }
+      ],
+      "rewards": [
+        {
+          "type": 3906625240,
+          "value": 1
+        }
+      ]
```
Comment (Iliasgx) [inline buildings/exocomp-factory.json:552]: Also parsing problem for rewards in level 15.

File: buildings/exocomp-factory.json [hunk 2]
```diff
@@ -0,0 +1,1824 @@
...
+          "name": "Operations",
+          "level": 35
+        }
+      ],
+      "rewards": [
+        {
+          "type": 1917231543,
+          "value": 1
+        }
+      ]
```
Comment (Iliasgx) [inline buildings/exocomp-factory.json:1277]: Parsing problem for level 35.

**Review by Iliasgx — CHANGES_REQUESTED** [review]
Could you check again for levels 15-24 if the second bonus isn't 3.0 instead of 2.0?

**Comment by WebSpider** [comment]
> Could you check again for levels 15-24 if the second bonus isn't 3.0 instead of 2.0?

Thanks, I missed that one!

File: buildings/exocomp-factory.json (hunk 1 above)
Comment (WebSpider) [reply buildings/exocomp-factory.json:552]: Yeah I'm resolving dependencies that apparently exist in rewards 🙈

**Comment by WebSpider** [comment]
> Could you check again for levels 15-24 if the second bonus isn't 3.0 instead of 2.0?

Checked data and in-game, both say grade 2, unlock 3

**Review by Iliasgx — APPROVED** [review]
Looks fine

**Pull request merged**